python/arcticdb/version_store/library.py

```diff
@@ -1383,9 +1383,12 @@ def finalize_staged_data(
         2024-01-03    3
         2024-01-04    4
         """
+        if not (mode is None or isinstance(mode, StagedDataFinalizeMethod) or mode == "WRITE" or mode == "APPEND"):
```

Comment:
`if not isinstance(mode, StagedDataFinalizeMethod) or mode not in ("write", "append")` is clearer to me, as it's closer to English